one_fm/one_fm/report/justification_report/justification_report.py

```diff
@@ -85,9 +86,11 @@ def set_day_total_overtime_attendance_check(cur_date, column_date_field, datapac
 def set_day_total_pending_approval_acheck(cur_date, column_date_field, datapack):
     pending_apporval = frappe.db.count('Attendance Check', {'date':cur_date, 'docstatus': ['<', 2], 'workflow_state': 'Pending Approval'})
     if len(datapack) == 3:
+        print("hey", pending_apporval)
```

Comment:
Remove the print

Reply:
@pjamsheer , changes have been implemented